send.go

```diff
@@ -34,10 +32,8 @@ func (f SendFunc) Send(from string, to []string, msg io.WriterTo) error {
 
 // Send sends emails using the given Sender.
 func Send(s Sender, msg ...*Message) error {
-	for i, m := range msg {
-		if err := send(s, m); err != nil {
-			return fmt.Errorf("gomail: could not send email %d: %v", i+1, err)
-		}
+	for _, m := range msg {
```

Comment:
Don't skip the errors returned by SMS providers!

Reply:
Okay，Thank you for your code review comment. Now I don't  skip the errors returned by SMS providers!